### ISSUE TITLE
server.Start(ip, Convert.ToInt32(txtPort.Text)); Null error

### ISSUE THREAD

**Issue by asenyurt** [BrandonPotter/SimpleTCP#68]
Hi when i click the start button its give me Null error from server.Start(ip, Convert.ToInt32(txtPort.Text));
ip is correct host is correct but i dont understand why i am getting error
![image](https://user-images.githubusercontent.com/73142096/100360980-4758c580-3002-11eb-99d3-1f09436fb4b0.png)

**Comment by Brhsoftco** [comment]
Each time the server starts, reset it:

```csharp
try {
  //validation checks
  if (string.IsNullOrWhitespace(txtHost.Text)) return;
  if (!int.TryParse(txtPort.Text, out var port)) return;

  //start requirements
  var ip = IPAddress.Parse(txtHost.Text);

  //IP validation check
  if (ip == null) return;

  //create the server
  var server = new SimpleTcpServer();

  //attempt server start
  server.Start(ip, port);
}
catch(Exception ex) {
  MessageBox.Show($"An error occurred whilst starting the server:\n\n{ex}");
}
```
You also don't need to directly specify an IP Address if you're positive the first valid adapter will be correct:
```csharp
//start the server by using the port only
server.Start(port);
```

**TL;DR: Always ensure you check for nulls and invalid values**
